apps/tenant-management-webapp/src/app/app.tsx

```diff
@@ -33,40 +33,23 @@ const AppRouters = () => {
         <h3>For the best experience, please use a Desktop</h3>
       </MobileMessage>
       <HideMobile>
-        <Router>
-          <Switch>
-            <Route exact path="/">
-              <LandingPage />
-            </Route>
+        <Routes>
+          <Route path="/" element={<LandingPage />} />
 
-            <Route path="/admin">
-              <PrivateApp>
-                <PrivateRoute path="/admin" component={Admin} />
-                <PrivateRoute path="/admin/tenants" component={TenantsRouter} />
-              </PrivateApp>
-            </Route>
-            <PublicApp>
-              <Route path="/:realm/login">
-                <Login />
-              </Route>
-              <Route path="/get-started">
-                <GetStarted />
-              </Route>
-              <Route exact path="/login-redirect">
-                <LoginRedirect />
-              </Route>
-              <Route exact path="/login-error">
-                <SignInError />
-              </Route>
-              <Route exact path="/logout-redirect">
-                <LogoutRedirect />
-              </Route>
-              <Route exact path="/tenant/creation">
-                <CreateTenant />
-              </Route>
-            </PublicApp>
-          </Switch>
-        </Router>
+          <Route path="/admin//*" element={<PrivateApp />}>
```

Comment:
is this path correct? There is an extra backslash in it?

Reply:
I tested it locally and with or without the /* does not affect the final results.